src/Workspaces/SharedUtilitiesAndExtensions/Compiler/CSharp/Formatting/CSharpFormattingOptions2.cs

```diff
@@ -5,26 +5,24 @@
 using System.Collections.Immutable;
 using System.Diagnostics;
 using Roslyn.Utilities;
+using Microsoft.CodeAnalysis.Options;
 
 #if CODE_STYLE
 using CSharpWorkspaceResources = Microsoft.CodeAnalysis.CSharp.CSharpCodeStyleResources;
 using WorkspacesResources = Microsoft.CodeAnalysis.CodeStyleResources;
-using Microsoft.CodeAnalysis.Internal.Options;
-#else
-using Microsoft.CodeAnalysis.Options;
 #endif
 
 namespace Microsoft.CodeAnalysis.CSharp.Formatting
 {
-    public static partial class CSharpFormattingOptions
+    internal static partial class CSharpFormattingOptions2
```

Comment:
Finally, one change that GitHub smartly recognized as file move + type rename instead of showing a new file addition!